app/lib/dartdoc/storage_path.dart

```diff
@@ -49,3 +50,13 @@ String contentObjectName(String packageName, String packageVersion, String uuid,
     String relativePath) {
   return p.join(contentPrefix(packageName, packageVersion, uuid), relativePath);
 }
+
+/// ObjectName of an SDK asset.
+String sdkObjectName(String relativePath) {
+  return p.join(_sdkAssetPrefix, relativePath);
+}
+
+/// The ObjectName for the Dart SDK's extracted dartdoc content.
+String dartSdkDartdocDataName(String runtimeVersion) {
+  return sdkObjectName('dart/pub-dartdoc-data/$runtimeVersion.json.gz');
```

Comment:
Does anything garbage collect these old files or do they accumulate indefinitely? Does this mean we have to update the runtime version whenever we update the Dart SDK? Should we decouple the Dart SDK used for the API results and the Dart SDK pub's website itself uses?

Reply:
> Does anything garbage collect these old files or do they accumulate indefinitely?

No GC yet, but there will be.

> Does this mean we have to update the runtime version whenever we update the Dart SDK? Should we decouple the Dart SDK used for the API results and the Dart SDK pub's website itself uses?

We do need to update it right now, but we could decouple the runtime SDK version from it. I'll need to think about it if there is any additional risk with the current stack, and I'll file a separate PR.